_pages/releases.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: "Apache Beam Releases"
+permalink: /releases/
+---
+
+# Apache Beam releases
+
+The easiest way to use Apache Beam is via one of the released versions in the
+[Maven Central Repository](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.beam%22).
+
+For example, if you are developing using Maven, add the following dependencies
```

Comment:
if you are developing using Maven and want to use the Java SDK with the DirectPipelineRunner, add...

(remove such as)